### DISCUSSION TITLE
Filter by topic name with TopicOperator

### DISCUSSION THREAD

**Discussion by obbiondo** [strimzi/strimzi-kafka-operator#7612] [Q&A] [answered]
Hello everyone,

I'm deploying the Strimzi Topic Operator as a standalone operator in my Kubernetes environment. When connecting with my external Kafka Cluster, all the topics metadata are retrieved from Zookeeper and saved in the topic store, as per documentation.

For some reason I don't want all my topics to be managed by the Topic Operator, but just some topics that follow a specific naming convention. 
Is it possible in the Topic Operator configuration to filter the topics to be managed by their (prefix) name?

Thanks

**Comment by scholzj** [comment, marked as answer]
No, there is currently no such filter available in the Topic Operator. It currently always handles all topics.

**Reply by obbiondo** [reply]
Do you think you will implement this feature?

**Reply by scholzj** [reply]
I have no idea. And I'm not sure it is completely easy with the bidirectional synchronization. You would probbaly need to filter them out in both directions. Plus it would be quite hard to decide what to do about the existing topics in existing installations.
